### PR TITLE
Lesson 10

### PR DESCRIPTION
**Pull request by vzuev401**
#3 is done I think.

### CODE REVIEW

**Comment by wowawowa** [comment]
hello

File: lesson_10/task_1.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from pathlib import Path
```
Comment (wowawowa) [inline lesson_10/task_1.py]: asd

**Comment by vzuev401** [comment]
it's ok?

**Comment by vzuev401** [comment]
hello?

**Pull request merged**